perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -103,6 +104,8 @@ def Configure(vm, seed_vms):
              'num_cpus': vm.num_cpus,
              'cluster_name': 'Test cluster'}
 
+  vm.UpdateVMHostsFile()
```

Comment:
I cloned your branch into my local machine. 
I am able to start Cassandra cluster without updating host file. 
What is the specific error message you see? You should be able to find logs in /tmp/pkb/apache-cassandra-2.1.13/logs/system.log